apps/system/js/utility_tray.js

```diff
@@ -97,13 +100,33 @@ window.UtilityTray = {
     Service.register('makeAmbientIndicatorInactive', this);
   },
 
+  /*
+   * The time after which we shouldn't use motion events to determine the
+   * speed of the utility tray opening/closing, in milliseconds.
+   */
+  VELOCITY_THRESHOLD: 50,
```

Comment:
nit: name is a little misleading, since this value is in ms an not px/ms. What about something like `SWIPE_TIME_THRESHOLD`?